### PR TITLE
327: git webrev: Print the location of the generated webrev upon successful completion

### PR DESCRIPTION
**Pull request by zhaosongzs**
Originally, git webrev command would return nothing when executed successfully. It is not a good experience for user. 

Right now, when executed successfully, the location of index.html will be printed.

Also, added a --quiet flag if the current silent behavior is required.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-327](https://bugs.openjdk.org/browse/SKARA-327): git webrev: Print the location of the generated webrev upon successful completion

### Reviewers
 * [Magnus Ihse Bursie](https://openjdk.org/census#ihse) (@magicus - **Reviewer**) ⚠️ Review applies to [d487b78b](https://git.openjdk.org/skara/pull/1379/files/d487b78b7917bd1df641e2ff127c193c79ea2af8)
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara pull/1379/head:pull/1379` \
`$ git checkout pull/1379`

Update a local copy of the PR: \
`$ git checkout pull/1379` \
`$ git pull https://git.openjdk.org/skara pull/1379/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1379`

View PR using the GUI difftool: \
`$ git pr show -t 1379`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1379.diff">https://git.openjdk.org/skara/pull/1379.diff</a>

</details>

### CODE REVIEW

**Comment by zhaosongzs** [comment]
https://api.github.com/users/zhaosongzs/orgs My orgs shows right currently. But it's still not work. So I think the change of my Username in GitHub is required.

**Review by magicus — APPROVED** [review]
Looks good to me.

File: cli/src/main/java/org/openjdk/skara/cli/GitWebrev.java [hunk 1]
```diff
@@ -454,6 +458,10 @@ private static void generate(String[] args) throws IOException {
                   .comments(comments)
                   .generate(base, head);
         }
+        if (!quiet) {
+            System.out.println("Webrev executed successfully, details are in the link below:");
+            System.out.println("file://" + new File(output.resolve("index.html").toUri()).getAbsoluteFile());
```
Comment (erikj79) [inline cli/src/main/java/org/openjdk/skara/cli/GitWebrev.java:463]: That looks like a weird way to generate a URL. Why not just:
```suggestion
            System.out.println(output.resolve("index.html").toUri());
```

Comment (zhaosongzs) [reply cli/src/main/java/org/openjdk/skara/cli/GitWebrev.java:463]: Yeah, will do the change

**Review by erikj79 — APPROVED** [review]

**Pull request closed without merging**